pkg/build/builder/sti.go

```diff
@@ -354,8 +362,9 @@ func buildEnvVars(build *api.Build) s2iapi.EnvironmentList {
 // the strategy's environment. There is no preference given to either lowercase
 // or uppercase form of the variable.
 func scriptProxyConfig(build *api.Build) (*s2iapi.ProxyConfig, error) {
-	httpProxy := ""
-	httpsProxy := ""
+	var httpProxy string
+	var httpsProxy string
```

Comment:
because i loath this way of misusing shorthand assignment for string variables.
when you define a new int do you do newInt := 0 ? do you do falseVar := false ?